src/gc/transaction_level_gc_manager.cpp

```diff
@@ -53,7 +53,10 @@ void TransactionLevelGCManager::Running(const int &thread_id) {
     auto &txn_manager = concurrency::TransactionManagerFactory::GetInstance();
     auto max_cid = txn_manager.GetMaxCommittedCid();
 
-    PL_ASSERT(max_cid != MAX_CID);
+    if (max_cid == MAX_CID) {
```

Comment:
I assume that the previous assert prevent some bad things from happening. In what case will max_cid == MAX_CID?